### PR TITLE
Adding support for FRRouting

### PR DESCRIPTION
**Pull request by pautiina**
https://frrouting.org/

### CODE REVIEW

**Comment by respawner** [comment]
The new `frr.php` file looks pretty much like the one for Quagga.

Maybe adding a `frr` case to the `routers.php` file could be enough?

**Comment by pautiina** [comment]
is a branch from Quagga, but have some some differences on the commands.
for example for Quagga:  `show ipv6 bgp regexp` - this command not working on FRRouting.
On FRRouting must be :  `show bgp ipv6 regexp`
FRRouting support outputting results as json. In the future, I think it is necessary to implement this. 
They also develop a new method of remote connection to the daemon.

File: routers/router.php [hunk 1]
```diff
@@ -188,6 +189,10 @@ public function send_command($command, $parameter) {
       case 'zebra':
         return new Quagga($config, $router_config, $id, $requester);
 
+      case 'frr':
+      case 'zebra':
```
Comment (respawner) [inline routers/router.php]: This will conflict with the first `zebra` case, which use the `Quagga` class.

Comment (pautiina) [reply routers/router.php]: Must be resolve now

**Pull request merged**